client/core/core.go

```diff
@@ -1125,18 +1125,12 @@ func (c *Core) Register(form *RegisterForm) (*RegisterResult, error) {
 func (c *Core) verifyRegistrationFee(wallet *xcWallet, dc *dexConnection, coinID []byte, assetID uint32) {
 	reqConfs := dc.cfg.RegFeeConfirms
 
-	regConfs, err := wallet.Confirmations(coinID)
-	if err != nil {
-		log.Errorf("Error getting confirmations for %s: %v", hex.EncodeToString(coinID), err)
-		return
-	}
-
-	dc.setRegConfirms(regConfs)
+	dc.setRegConfirms(0)
```

Comment:
The initial confirmations might not be zero if called from `reFee`. The caller does know the number of current confirmations, though (`0` for `Register`, `confs` for `reFee`), so you could add initial confirmations as a parameter to `verifyRegistrationFee`.